app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
+    current_time = Time.zone.now
+
+    distance_of_time_in_words(
+      current_time, current_time + lockout_time_remaining, true, highest_measures: 2
+    )
+  end
+
+  def lockout_time_remaining
+    (Devise.direct_otp_valid_for - (Time.zone.now - Time.zone.now)).to_i
```

Comment:
Yeah this is just a placeholder for now...should have just thrown it in the controller but was trying to get clever. Given that we have these methods in `UserDecorator` but also want the same logic when a user is not logged in, where do you think these methods belong?